docs/source/releases/v1_12_2a0.rst

```diff
@@ -0,0 +1,58 @@
+ | # # # Distribution Statement A. Approved for public release. Distribution unlimited.
+ | # # #
+ | # # # Author:
+ | # # # Naval Research Laboratory, Marine Meteorology Division
+ | # # #
+ | # # # This program is free software: you can redistribute it and/or modify it under
+ | # # # the terms of the NRLMMD License included with this program. This program is
+ | # # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+ | # # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+ | # # # for more details. If you did not receive the license, for more information see:
+ | # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+Version 1.12.2a0 (2024-02-21)
+*****************************
+
+* Bug fixes
+
+  * Ensure that log.interactive works from plugins when imported independently
```

Comment:
Isn't this doing a lot more than moving the interactive log level to __init__?  Did you update the whole log_setup to be class-based within this PR?

Reply:
Yes, yes I did. I'll update this.

Reply:
Updated.